### PR TITLE
LPD-29368 Remove @Ignore annotation for LayoutExportImportTest.testFriendlyURLCollision

### PR DESCRIPTION
**Pull request by balazssk**
https://liferay.atlassian.net/browse/LPD-27254

fixed ticket number in commit message

ref https://github.com/liferay-echo/liferay-portal/pull/15840

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java [hunk 1]
```diff
@@ -605,6 +625,10 @@ public void testFriendlyURLCollision() throws Exception {
 			layoutA.getUserId(), layoutA.getPlid(), friendlyURLB + "-de", "de");
 
 		exportImportLayouts(layoutIds, getImportParameterMap());
+
+		_assertNewFriendlyURL(layoutA, friendlyURLB);
+
```
Comment (ealonso) [inline modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java:630]: @balazssk you should remove this empty line

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java [hunk 2]
```diff
@@ -584,19 +585,38 @@ public void testFriendlyURLCollision() throws Exception {
 
 		exportImportLayouts(layoutIds, getImportParameterMap());
 
+		FriendlyURLEntry friendlyURLEntry =
```
Comment (ealonso) [inline modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java:588]: @balazssk you should move it closer the usage

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java [hunk 3]
```diff
@@ -989,6 +1013,24 @@ protected void testAvailableLocales(
 		}
 	}
 
+	private void _assertNewFriendlyURL(Layout layoutA, String friendlyURLB)
```
Comment (ealonso) [inline modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java:1016]: @balazssk you don't need to add A or B to parameters

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java (hunk 2 above)
Comment (balazssk) [reply modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java:594]: fixed

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java (hunk 3 above)
Comment (balazssk) [reply modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java]: fixed

**Comment by balazssk** [comment]
ci:test:sf

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java [hunk 4]
```diff
@@ -989,6 +1012,24 @@ protected void testAvailableLocales(
 		}
 	}
 
+	private void _assertNewFriendlyURL(Layout layout, String friendlyURL)
+		throws Exception {
+
+		Layout importedLayoutA = _layoutLocalService.getLayoutByUuidAndGroupId(
```
Comment (ealonso) [inline modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java:1018]: @balazssk last change this variable doesn't need the A

Comment (balazssk) [reply modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/test/LayoutExportImportTest.java]: fixed

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**